### ISSUE TITLE
Migrate and archive old attachments

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#4298]

**Comment by amenity** [comment]
@johnclary @mateoclarke - potentially something Jorge could work on? We've had committed to do this as a "kick off the new year with a security sprint" epic that we never got to. 🙈

**Comment by johnclary** [comment]
Yes, although onboarding to the Knack services ecosystem is not small. Up to y'all.

**Comment by ChrispinP** [comment]
@johnclary Is this something we still plan on doing?

**Comment by johnclary** [comment]
Leaving this in backlog for now. Hopefully we can just deprecate the entire app once the Passport solution is up
